vald/evm/evm_test.go

```diff
@@ -211,7 +211,7 @@ func TestMgr_GetTxReceiptIfFinalized(t *testing.T) {
 		Then("tx is considered not finalized", func(t *testing.T) {
 			txReceipt, err := mgr.GetTxReceiptIfFinalized(chain, tx.Hash(), confHeight)
 
-			assert.ErrorContains(t, err, "failed")
+			assert.NoError(t, err)
 			assert.Nil(t, txReceipt)
```

Comment:
maybe change this test as well
https://github.com/axelarnetwork/axelar-core/pull/2100/commits/d5b4dfcbbefc4631925446094f97c07f26c0ac97#diff-e2f9081c23af243664ab48fe38000b488227e2309f12696ede9da0254115bedcR923

Reply:
No. Despite the fact that these 2 functions are named the same way, they do not behave remotely the same at all.